pkg/core/native/policy.go

```diff
@@ -332,6 +348,41 @@ func (p *Policy) IsBlockedInternal(dao dao.DAO, hash util.Uint160) bool {
 	return dao.GetStorageItem(p.ContractID, key) != nil
 }
 
+func (p *Policy) getStoragePrice(ic *interop.Context, _ []stackitem.Item) stackitem.Item {
+	return stackitem.NewBigInteger(big.NewInt(p.GetStoragePriceInternal(ic.DAO)))
+}
+
+// GetStoragePriceInternal returns current execution fee factor.
+func (p *Policy) GetStoragePriceInternal(d dao.DAO) int64 {
+	p.lock.RLock()
+	defer p.lock.RUnlock()
+	if p.isValid {
+		return int64(p.storagePrice)
+	}
+	return int64(getUint32WithKey(p.ContractID, d, storagePriceKey, StoragePrice))
+}
+
+func (p *Policy) setStoragePrice(ic *interop.Context, args []stackitem.Item) stackitem.Item {
+	value := toBigInt(args[0]).Int64()
```

Comment:
`toUint32(...)` function from `native_nep17.go` can be used here. This function can also be moved to `util.go` file.

Reply:
The same thing can be done for `ExecFeeFactor`.

Reply:
Fixed.